diff --git a/NNDCheckIn/NNDCheckIn.xcodeproj/project.pbxproj b/NNDCheckIn/NNDCheckIn.xcodeproj/project.pbxproj
index 0d45f3d..38b771f 100644
--- a/NNDCheckIn/NNDCheckIn.xcodeproj/project.pbxproj
+++ b/NNDCheckIn/NNDCheckIn.xcodeproj/project.pbxproj
@@ -365,7 +365,7 @@
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
-				SYSTEM_HEADER_SEARCH_PATHS = "\"~/Documents/git/NNDDataBase/NNDDataBase\"/**";
+				SYSTEM_HEADER_SEARCH_PATHS = "\"~/Documents/git/NNDDataBase/NNDDataBase\"/** \"~/Documents/git/NNDCommon/NNDCommon\"";
 				TARGETED_DEVICE_FAMILY = "1,2";
 				VERSIONING_SYSTEM = "apple-generic";
 				VERSION_INFO_PREFIX = "";
@@ -411,7 +411,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				SDKROOT = iphoneos;
-				SYSTEM_HEADER_SEARCH_PATHS = "\"~/Documents/git/NNDDataBase/NNDDataBase\"/**";
+				SYSTEM_HEADER_SEARCH_PATHS = "\"~/Documents/git/NNDDataBase/NNDDataBase\"/** \"~/Documents/git/NNDCommon/NNDCommon\"";
 				TARGETED_DEVICE_FAMILY = "1,2";
 				VALIDATE_PRODUCT = YES;
 				VERSIONING_SYSTEM = "apple-generic";
diff --git a/NNDCheckIn/NNDCheckIn/Controllers/NNDCheckInController.m b/NNDCheckIn/NNDCheckIn/Controllers/NNDCheckInController.m
index cea2338..9468830 100644
--- a/NNDCheckIn/NNDCheckIn/Controllers/NNDCheckInController.m
+++ b/NNDCheckIn/NNDCheckIn/Controllers/NNDCheckInController.m
@@ -9,6 +9,7 @@
 #import <Foundation/Foundation.h>
 #import "NNDCheckInController.h"
 #include "NNDDataBaseController.h"
+#include "NNDPatientDataModel.h"
 
 //NNDCheckInService
 // Extension method add private protected items into it
@@ -50,6 +51,11 @@ - (instancetype)initWithDataModel {
 
         [_nndCheckInService nndCheckInUIViewInitializer:_model ];
         [ _dataBaseController HelloDataBaseController];
+        // init data model for Patient DataModel,
+        NNDPatientDataModel *patientDataModel = [ _dataBaseController NNDPatientDataModelInit];
+
+        // [ _dataBaseController PatiendModelData init:WithUserId];
+
 
         /*
          Author: Kian D.Rad
@@ -59,8 +65,6 @@ - (instancetype)initWithDataModel {
             1- Build UIView, 
             2- Build button
 
-         
-         
          PatiendDataMode  = [[PatiendDataMode alloc] initDataModel];
          LocationOfUser   = [[LocationOfUser alloc] initWithDataModel];
          StatusOfVisit    = [[statusOfVisit alloc] initWithDataModel];
